src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.ts

```diff
@@ -131,9 +133,15 @@ export class PoolFormComponent extends CdForm implements OnInit {
       minBlobSize: new UntypedFormControl('', {
         updateOn: 'blur'
       }),
+      minBlobSizeUnit: new UntypedFormControl(this.blobUnits[0], {
+        updateOn: 'blur'
+      }),
       maxBlobSize: new UntypedFormControl('', {
         updateOn: 'blur'
       }),
+      maxBlobSizeUnit: new UntypedFormControl(this.blobUnits[0], {
```

Comment:
```suggestion
      maxBlobSizeUnit: new UntypedFormControl(this.blobUnits[2], {
```
default MiB, based on above changes.